.travis.yml

```diff
@@ -6,10 +6,20 @@ node_js:
 - 8 # EOL: December 2019
 - 6 # EOL: April 2019
 
+os:
+  - linux
+
 matrix:
   fast_finish: true
   include:
-    - node_js: 8 # Version used to deploy to npm registry
+    - node_js: 8
+      os: windows
+      env: YARN_GPG=no # starts gpg-agent that never exits
```

Comment:
`YARN_GPG=no` is needed because travis does not kill all processes on build finish (installing yarn starts gpg-agent). I dont know if they will fix it, but we will be able to remove this env if they do.